tools/census_contrib/embedding_metadata.md

```diff
@@ -21,6 +21,7 @@ Each embedding will contain a variety of metadata stored in the SOMA `metadata`
 | Field name             | Required | Type          | Description                                                                           |
 | ---------------------- | -------- | ------------- | ------------------------------------------------------------------------------------- |
 | id                     | required | string        | CZI-assigned accession ID for this embedding                                          |
+| embedding_name         | required | string        | Name of the algorithm used to generate the embedding                                  |
```

Comment:
If we keep this as `required`, we should also modify the metadata in the existing embeddings, other than `contributions.json`. Otherwise, we can leave this as optional and only define it where we needed. Easier but not as clean. @bkmartinjr thoughts?

Reply:
I would make it required for simplicity.  it is also quite easy to manually patch the array metadata if that is desired.

I would add to the schema a note that this value is NOT guaranteed to be unique across all embeddings (only the ID is guaranteed unique).